### PR TITLE
FIX geojson icon and lessc compile issue

### PR DESCRIPTION
**Pull request by camfindlay**
Fixes the missing geojson icon and a lessc compiling issue with Bootstrap 2. See https://stackoverflow.com/questions/26628309/less-v2-does-not-compile-twitters-bootstrap-2-x

### CODE REVIEW

File: ckanext/dia_theme/fanstatic/dia_custom.css [hunk 1]
```diff
@@ -7876,6 +7876,10 @@ h4 small {
   height: 35px;
   background-position: -288px -62px;
 }
+.format-label[data-format=geojson],
```
Comment (camfindlay) [inline ckanext/dia_theme/fanstatic/dia_custom.css]: Change here for background pos override. lessc osx updates many of the hex strings.

File: ckanext/dia_theme/less/icons.less [hunk 2]
```diff
@@ -145,6 +145,12 @@
   .ckan-icon-background-position(9, "formatMedium");
 }
 
+//Fix to display geojson icon
+.format-label[data-format=geojson],
+.format-label[data-format*=geojson] {
+  background-position: -128px -62px;
```
Comment (dythya) [inline ckanext/dia_theme/less/icons.less]: why not use the  .ckan-icon-backgro....
as is used for the other icons?

Comment (camfindlay) [inline ckanext/dia_theme/less/icons.less]: The stylesheet from the geo extension appears to override things, these seemed to be the min change to achieve the result.

Comment (dythya) [inline ckanext/dia_theme/less/icons.less]: ok

**Pull request merged**